Library/Homebrew/cmd/update-report.rb

```diff
@@ -250,10 +250,23 @@ def migrate_tap_migration
       tabs = dir.subdirs.map { |d| Tab.for_keg(Keg.new(d)) }
       next unless tabs.first.tap == tap # skip if installed formula is not from this tap.
       new_tap = Tap.fetch(new_tap_name)
-      new_tap.install unless new_tap.installed?
-      # update tap for each Tab
-      tabs.each { |tab| tab.tap = new_tap }
-      tabs.each(&:write)
+      # related to formulae deduplication. If formula stays in cask, autoinstall or provide cask instructions
+      if new_tap_name == "caskroom/cask"
+        if new_tap.installed? && File.exist?("/usr/local/Caskroom")
```

Comment:
They've recently migrated as far as I understood and have usr/local as new default path
https://github.com/caskroom/homebrew-cask/issues/21913